build.gradle.kts

```diff
@@ -31,5 +34,11 @@ tasks {
     }
     test {
         useJUnitPlatform()
+
+        dependsOn("cleanTest")
+
+        testLogging {
+            events("passed", "skipped", "failed")
+        }
```

Comment:
Can't understand why do you need this?

Reply:
just for easier tracking test exceptions in the future. If you think, that it is irrelevant - I can delete it